code/modules/power/apc.dm

```diff
@@ -767,15 +767,19 @@
 		wires.Interact(user)
 		return	//The panel is visibly dark when the wires are exposed, so we shouldn't be able to interact with it.
 
-	return nano_ui_interact(user)
+	return ui_interact(user)
```

Comment:
idk if this is still needed as attack hand calls tgui

Reply:
attck_hand calles the interact proc, which then calls the ui_interact